powermetrics_compare.py

```diff
@@ -97,8 +100,11 @@ def Summary(results, filename):
     scenario_result = results[scenario].drop(0)
     nanoseconds_to_seconds = 1000000000.0
     scenario_result['elapsed_s'] = scenario_result['elapsed_ns'] / nanoseconds_to_seconds
-
     scenario_result[rate_columns] = scenario_result[rate_columns].mul(scenario_result['elapsed_s'], axis=0)
+    
+    scenario_result['charge_delta'] = scenario_result['charge_delta'].fillna(0)
+    scenario_result = scenario_result[(np.abs(stats.zscore(scenario_result[['elapsed_s', 'charge_delta', 'package_joules']])) < 3).all(axis=1)]
```

Comment:
Can you please leave a big nice comment saying why we are filtering here please?

Also can we add a column that's like cumulative discharge to do what we talked about and be able to analyze the total discharge that happened over the whole test?

Reply:
Done. I added a column that the total discharge (first - last charge, which also corresponds to sum(delta))